### ISSUE TITLE
networked libraries?

### ISSUE THREAD

**Issue by PBiM** [tattwamasi/tattwamasi.github.io#1]
I keep iTunes on an older Mac running 10.7.5 and it's networked to my laptop running 10.11 (beta). Running TT on the laptop lets me choose a music folder on the networked drive but the playlists are from the local iTunes library (nearly empty) and no files are scanned when tested.
![screen shot 2015-08-05 at 1 05 23 pm](https://cloud.githubusercontent.com/assets/13663930/9092209/b1080b1a-3b72-11e5-8026-a958446a8796.jpg)
![screen shot 2015-08-05 at 1 05 37 pm](https://cloud.githubusercontent.com/assets/13663930/9092210/b12a3708-3b72-11e5-9ef4-bff63cd873df.jpg)
Will it be possible to use a networked iTunes library in this way?

**Comment by tattwamasi** [comment]
Yes, I think you can do what you intended.  
tl;dr:  If you want playlists from a different iTunes library, use option/alt when starting iTunes and select the iTunes library you want so that the app can see the right playlists.  Double-check in iTunes that it can play items from the newly selected library.

Longer explanation:
First thing - Consider the two copy choices as completely independent.  In your screenshot (which helps tremendously is seeing what's going on) you don't have the Copy whole Library/Folder selected (no checkmark).  So though you chose what folder to do at some point, as of the time of the screenshot, it wouldn't actually do it.  Go ahead and select it and it will do the copy of everything in that folder.

"But I didn't want everything in that folder and subfolders, just certain things in specific playlists!", you may be thinking.  If that's the case, then that's all controlled by the "Copy Playlist(s)" checkmark, and has nothing to do with the first option (you don't even have to select a folder).    

"But if it's completely independent, how does it know where to get the files from then?" Same way it knew what the playlists were in the first place…  Apple has a library I use to interface with iTunes.  It tells me where the current iTunes folder is, playlist info, etc., and when I look at the playlist each item has it's location information.

So.. sounds like you are saying it's looking at the wrong iTunes library info right?  I think if you hold down option/alt while starting iTunes it'll ask you to select the library you want.  If you do this before running this app, then when it gets the iTunes library and playlist info, it should get the info you want.  After finishing up, you can switch the active library back by restarting iTunes with alt/option held down and reselecting the original library.

Hope that helps.

**Comment by PBiM** [comment]
Works. Simple (on my end, not really on the back end). Thanks, sorry to bother with something that should have been obvious (that it uses the known iTunes library data).
Getting ready for the ‘X

**Comment by tattwamasi** [comment]
Awesome, glad things work.  Hopefully there'll be some improvements or new surprise feature in the media interface for the X (and S, as sibling models) by then!
